build.gradle.kts

```diff
@@ -112,6 +112,7 @@ dependencies {
     testImplementation("org.testcontainers:r2dbc:1.17.3")
     testImplementation("org.testcontainers:mysql:1.17.3")
     testImplementation("org.awaitility:awaitility:4.2.0")
+    testImplementation("org.jsoup:jsoup:1.15.3")
```

Comment:
Is it possible to declare this dependency as for testing only?

Reply:
as I understand `testImplementation()` adds test dependency and `implementation()` for compile and runtime...

Reply:
Gee, I think you are right. I overlooked that part.